.github/workflows/black.yml

```
@@ -9,11 +10,9 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: black
-        uses: lgeiger/black-action@v1.0.1
-        with:
-          args: .
+        uses: psf/black@stable
```

```suggestion
        uses: psf/black@stable
        with:
          options: "--check --verbose"
```
Checkout https://black.readthedocs.io/en/stable/integrations/github_actions.html#usage:
> Please note that the [--check flag](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#labels-exit-code) is required so that the workflow fails if Black finds files that need to be formatted.